testsuite/examples/test_framework_GRASS_GIS_with_NC.conf

```diff
@@ -27,3 +27,15 @@ REPORTS="testreports"
 PUBLISH="no"
 # upload WWW dir on server for report publication (not used for local tests)
 SERVERDIR="/var/www/html/grassgistestreports"
+
+# read module tests list from file
+# may be no|yes
+READ_MODULE_TEST_LIST=yes
```

Comment:
Is it wise to default this to `yes`? 
I think not unless you also commit the `test.list` file.
Of course, the contents of this file will need to be maintained too.

Reply:
Absolutely right, I changed it to 'no'